rabies/analysis_pkg/cpca/report.py

```
@@ -0,0 +1,230 @@
+    
+    R2_s = SSnet_s_loss/(SSnet_s_loss.T+SSe_loss).T
+    R2_t = SSnet_t_loss/(SSnet_t_loss.T+SSe_loss).T
+    
+    return Snet_s_l,Snet_t_l,prior_sim_l,Dc_Cnet,Dc_Wnet,R2_s,R2_t
+
+
+def optim_n(prior_sim_l,min_prior_sim,Dc_Cnet,Dc_Wnet,Dc_W_thresh,Dc_C_thresh):
+    if (min_prior_sim is None) and (Dc_W_thresh is None) and (Dc_C_thresh is None):
+        return None
+    
+    if not min_prior_sim is None:
```

Flake8 (7.2.0)

[error] test for object identity should be 'is not'

(E714)